3rdParty/fuerte/src/H1Connection.h

```diff
@@ -146,6 +151,11 @@ class H1Connection final : public fuerte::GeneralConnection<ST> {
   // We use this to compute the timeout of the corresponding read if the
   // write was done.
 
+  std::atomic<int>
+      _leased;  // set to 1 by the `lease` method, prevents idle alarm if
```

Comment:
`_leased` is nowhere initialized. We should initialize it to `0` in the constructor.

Reply:
Done.